src/staff-handbook/pay-pension-and-benefits/pension.md

```diff
@@ -3,15 +3,10 @@ title: Pension
 related_order: 2
 last_reviewed_at: ""
 ---
+dxw provides a Pension Salary Exchange (PSE) scheme which is operated by Aviva. In a PSE scheme, contributions are deducted from your gross salary, resulting in a saving on tax and National Insurance Contributions (NICs). 
 
-dxw provides a pension which is operated by Aviva. Once you have passed your 3-month check in, you will be auto enrolled into the scheme, receive details by
-post of the necessary employer/employee contributions and have the option to set
-your percentage or opt out.
+In a PSE scheme, dxw will pay all pension contributions into your pension scheme, meaning a higher amount of employer contributions. The increase in employer contributions is then offset by an equivalent reduction in gross salary. dxw takes the salary you have exchanged and pays that straight into your pension fund.
 
-dxw will match any contribution up to 5%.
+Once you have passed your 3 month check in, you will be auto-enrolled into the scheme and the People team will be in touch with some more details and the option to set your contribution %.
```

Comment:
```suggestion
Once you have passed your 3 month check in, you will be auto-enrolled into the scheme. The People team will be in touch with more details and the option to set your contribution percentage.
```

Not sure, but the word "percentage" might be better for screen readers than the symbol

Reply:
this is unchanged from the the previous playbook content

Reply:
It's currently "percentage": https://github.com/dxw/playbook/blob/main/src/staff-handbook/pay-pension-and-benefits/pension.md?plain=1#L9